Source/DataSources/PropertyBag.js

```diff
@@ -10,8 +10,8 @@ import Property from "./Property.js";
  * A {@link Property} whose value is a key-value mapping of property names to the computed value of other properties.
  *
  * @alias PropertyBag
+ * @implements Record<string, any>
```

Comment:
The built in [Record utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype) is a generic that represents the same thing as the custom interface previously defined for `DictionaryLike`.  As a Typescript user this is very clear to me but I don't want to make it less clear to non-TS users.  (In fairness, I think "Record" is a common name for this pattern...)